guides/release/environment/contents.md

```diff
@@ -3,9 +3,10 @@ A Cardstack environment is the context that Cards run in. By default, it contain
 
 - an application that Cards render in
 - An API that receives requests from the front end, persists changes, and indexes data from external resources
-- An ephemeral data cache so that requests have speedy responses
+- An data cache so that requests have speedy responses
```

Comment:
We cache built-in cards in the browser, is that what you mean here? otherwise I'm not too sure what this is in reference to

Reply:
What I'm referring to is, it takes a long time to load data from git, so we used to cache this information for quick access, or so I thought. Was that removed in v2?

Reply:
So that was never that rationale. We in the past cached time shield the user from over the wire calls—but data _always_ loaded from the index. The only time you waited on github was for writes. 

Such a cache doesn’t really exist anymore. We do cache built in cards because by their nature they never change. But there is no more local store. Eventually orbit will provide that though.